OpenHPL/Waterway/SurgeTank.mo

```diff
@@ -53,6 +54,7 @@ model SurgeTank "Model of the surge tank/shaft"
   SI.Force F_g "Gravity force";
   SI.Pressure p_t "Pressure at top of the surge tank";
   SI.Pressure p_b "Pressure at bottom of the surge tank";
+  SI.Pressure p_n "Pressure at node or manifold of the surge tank";
```

Comment:
Is it really necessary to introduce a new variable just to link it directly to  an existing one. Why not simply do `p_b = i.p` which would would achieve the same but with one less variable. Correct?